x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/utils/highlighted_fields_config.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const highlightedFieldsPrefixToExclude = ['kibana.alert.rule', 'signal.rule', 'rule'];
```

Comment:
can you please add a comment why do we need to exclude these fields? Thanks

Reply:
please find [d6da91a](https://github.com/elastic/kibana/pull/159029/commits/d6da91ac9d21a620aa0b8246af087e06cabb2f65)